tinkerers/Fathima.md

```diff
@@ -0,0 +1,6 @@
+# Fathima Firoz
+   Hy there , I Fathima Firoz pursuing Btech in computer science and cyber security at Ilahia engineering college.
+   My goal is to learn and develop new skills in coding.
+
+# Linkedin 
+https://www.linkedin.com/in/fathima-firoz-0b6b6222b
```

Comment:
Heyy Fathima, please use proper markdown syntax to display this URL as a hyperlink.

```
like this 👇
[Linkedin](YOUR_PROFILE_URL)
```